### DISCUSSION TITLE
GitHub App for a GitHub Organization didn't create onboarding PRs after changing to interactive mode

### DISCUSSION THREAD

**Discussion by kaltepeter** [renovatebot/renovate#29701] [Request Help]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

I have added the GitHub App to an organization and selected 'All Repositories', it took me a while to find https://docs.renovatebot.com/config-overview/#onboarding-behavior and realize that it wasn't onboarding because all repositories are in silent mode. 

I have gone into Renovate at the organization level and changed from 'silent' to 'interactive' mode. No onboarding PRs have opened. 

It has now been over a couple days and no onboarding PR. I tried turning it off at the organization level and re-enabling and still no onboarding pr.

Currently my org level setting is interactive and each repo in renovate thinks it should be interactive but the state is silent and no onboarding prs are opened.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Are there jobs running for the repos, or not?

If the jobs are running but they still think they're in silent mode then that's one type of problem. If they're not even being enqueued then that's a different type of problem.

Either way, could you try suspending and unsuspending your installation to see if that will trigger the jobs with correct state?

**Reply by kaltepeter** [reply]
@rarkins The jobs weren't running. I just tried suspending and unsuspending and nothing happened.

**Reply by rarkins** [reply]
Can you try waiting 5-10 minutes after suspending before unsuspending? If this doesn't work, please try uninstalling, waiting 5 minutes, then reinstalling.

We have a bug on the backend to fix this properly - I'm just trying to help you work around it in the short term
